envs/example/defaults-2.0.yml

```diff
@@ -646,6 +646,9 @@ serverspec:
 inspec:
   enabled: False
 
+horizon:
+  customize: false
```

Comment:
since our default is false we should default the checks to false with `|default('false')` that way we don't need it unless its been explicitly set in envs in which case it will be avaiable.